doc/nrf/working_with_nrf/nrf53/thingy53.rst

```diff
@@ -295,37 +290,17 @@ FOTA updates on the Nordic Thingy:53 are supported using MCUmanager's Simple Man
 The application acts as a GATT server and allows the connected Bluetooth Central to perform the firmware update for both the application and network core.
 The Bluetooth configuration is updated to allow quick DFU data transfer.
 
-The application supports SMP handlers related to:
-
-* Image management.
-* Operating System (OS) management -- used to reboot the device after the firmware upload is complete.
-* Erasing settings partition -- used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
```

Comment:
I am not sure why this section is removed

Reply:
Removed that as I moved that to section in nRF52/53 to not duplicate docs.